lib/adapters/REST/endpoints/resource-type.ts

```diff
@@ -0,0 +1,46 @@
+import type { RawAxiosRequestHeaders } from 'axios'
+import type { AxiosInstance } from 'contentful-sdk-core'
+import * as raw from './raw'
+import copy from 'fast-copy'
+import type { CollectionProp } from '../../../common-types'
+import { type GetResourceProviderParams, type GetResourceTypeParams } from '../../../common-types'
+import type { RestEndpoint } from '../types'
+import type { ResourceTypeProps, UpsertResourceTypeProps } from '../../../entities/resource-type'
+
+const getBaseUrl = (params: GetResourceTypeParams) =>
```

Comment:
shouldn't the base URL not include the resource ID?

Reply:
looking at the other examples in the same directory, you are right, the base url should be the one without the resourceId and then you have a separate method for the other url.